input/ez.raml

```diff
@@ -2120,109 +2120,109 @@ version: 2.5
                         description: Moved permanently.
         /{path}:
             get:
                 displayName: Load User Group
-                description: Loads user groups for the given {path}
+                description: Loads User Groups for the given {path}.
                 headers:
                     Accept:
-                        description: If set the new user group is returned in XML or JSON format
+                        description: If set, the new User Group is returned in XML or JSON format.
                         example: |
                             application/vnd.ez.api.UserGroup+xml
                             application/vnd.ez.api.UserGroup+json
                     If-None-Match:
                         description: ETag
                 responses:
                     200:
-                        description: OK - loads user groups
+                        description: OK - loads User Groups.
                         body:
                             application/vnd.ez.api.UserGroup+xml:
                                 type: UserGroup
                                 example: !include examples/user/groups/path/GET/UserGroup.xml.example
                     401:
-                       description: Error - the user has no permission to read user groups
+                       description: Error - the user has no permission to read User Groups.
                     404:
-                       description: Error -	if the user group does not exist
+                       description: Error -	the User Group does not exist.
             patch:
                 displayName: Update User Group
-                description: Updates a user group - PATCH or POST with header X-HTTP-Method-Override PATCH
+                description: Updates a User Group. PATCH or POST with header X-HTTP-Method-Override PATCH.
                 headers:
                     Accept:
-                        description: If set the new user group is returned in XML or JSON forma
+                        description: If set, the new User Group is returned in XML or JSON format.
                         example:  |
                             application/vnd.ez.api.UserGroup+xml
                             application/vnd.ez.api.UserGroup+json
                     Content-Type:
-                        description: The UserGroupUpdate schema encoded in XML or JSON
+                        description: The UserGroupUpdate schema encoded in XML or JSON format.
                         example:  |
                             application/vnd.ez.api.UserGroupUpdate+json
                             application/vnd.ez.api.UserGroupUpdate+xml
                     If-Match:
-                        description: Causes to patch only if the specified ETag is the current one. Otherwise a 412 is returned.
+                        description: Causes to PATCH only if the specified ETag is the current one. Otherwise a 412 is returned.
```

Comment:
```suggestion
                        description: Performs the PATCH only if the specified ETag is the current one. Otherwise a 412 is returned.
```